package.json

```diff
@@ -67,7 +67,8 @@
     "sass-loader": "^10.1.0",
     "typescript": "^4.0.3",
     "webpack": "^5.36.2",
-    "webpack-cli": "^4.7.0"
+    "webpack-cli": "^4.7.0",
+    "d3-selection": "^3.0.0"
```

Comment:
Ah, right, about this one: Ideally we don't add this dependency here. I need to talk to Florian about this in general (#320 is still outstanding due to the same issue).

For the moment though, can you successfully build/run everything without adding this dependency? All the other PRs so far seem to function well without it.